.travis.yml

```diff
@@ -1,36 +1,92 @@
+  - BROWSER="chrome" FEATURE="salad/features/steps-browser-browsers.feature" REMOTE_ARG="http://${SAUCE_USERNAME}:${SAUCE_ACCESS_KEY}@ondemand.saucelabs.com:80/wd/hub" PLATFORM="OS X 10.6"
+  - BROWSER="chrome" FEATURE="salad/features/steps-browser-elements.feature" REMOTE_ARG="http://${SAUCE_USERNAME}:${SAUCE_ACCESS_KEY}@ondemand.saucelabs.com:80/wd/hub" PLATFORM="OS X 10.6"
+  - BROWSER="chrome" FEATURE="salad/features/steps-browser-forms.feature" REMOTE_ARG="http://${SAUCE_USERNAME}:${SAUCE_ACCESS_KEY}@ondemand.saucelabs.com:80/wd/hub" PLATFORM="OS X 10.6"
+  - BROWSER="chrome" FEATURE="salad/features/steps-browser-general.feature" REMOTE_ARG="http://${SAUCE_USERNAME}:${SAUCE_ACCESS_KEY}@ondemand.saucelabs.com:80/wd/hub" PLATFORM="OS X 10.6"
+  - BROWSER="chrome" FEATURE="salad/features/steps-browser-javascript.feature" REMOTE_ARG="http://${SAUCE_USERNAME}:${SAUCE_ACCESS_KEY}@ondemand.saucelabs.com:80/wd/hub" PLATFORM="OS X 10.6"
+  - BROWSER="chrome" FEATURE="salad/features/steps-browser-navigation.feature" REMOTE_ARG="http://${SAUCE_USERNAME}:${SAUCE_ACCESS_KEY}@ondemand.saucelabs.com:80/wd/hub" PLATFORM="OS X 10.6"
+  - BROWSER="chrome" FEATURE="salad/features/steps-browser-ordinals.feature" REMOTE_ARG="http://${SAUCE_USERNAME}:${SAUCE_ACCESS_KEY}@ondemand.saucelabs.com:80/wd/hub" PLATFORM="OS X 10.6"
+  - BROWSER="chrome" FEATURE="salad/features/steps-browser-page.feature" REMOTE_ARG="http://${SAUCE_USERNAME}:${SAUCE_ACCESS_KEY}@ondemand.saucelabs.com:80/wd/hub" PLATFORM="OS X 10.6"
+  - BROWSER="chrome" FEATURE="salad/features/steps-browser-remember.feature" REMOTE_ARG="http://${SAUCE_USERNAME}:${SAUCE_ACCESS_KEY}@ondemand.saucelabs.com:80/wd/hub" PLATFORM="OS X 10.6"
+  # no local chrome! there are problems with chrome on travis.
+  # local PHANTOMJS can run all tests except alerts.feature and some mouse actions (mouse-not-phantomjs.feature)
+  - BROWSER="phantomjs" FEATURE="salad/other_features/steps-browser-drag.feature"
+  - BROWSER="phantomjs" FEATURE="salad/other_features/steps-browser-mouse.feature"
+  # ---
+  - BROWSER="phantomjs" FEATURE="salad/features/steps-browser-blur.feature"
+  - BROWSER="phantomjs" FEATURE="salad/features/steps-browser-browsers.feature"
+  - BROWSER="phantomjs" FEATURE="salad/features/steps-browser-elements.feature"
+  - BROWSER="phantomjs" FEATURE="salad/features/steps-browser-forms.feature"
+  - BROWSER="phantomjs" FEATURE="salad/features/steps-browser-general.feature"
+  - BROWSER="phantomjs" FEATURE="salad/features/steps-browser-javascript.feature"
+  - BROWSER="phantomjs" FEATURE="salad/features/steps-browser-navigation.feature"
+  - BROWSER="phantomjs" FEATURE="salad/features/steps-browser-ordinals.feature"
+  - BROWSER="phantomjs" FEATURE="salad/features/steps-browser-page.feature"
+  - BROWSER="phantomjs" FEATURE="salad/features/steps-browser-remember.feature"
 install:
-  - "python setup.py install"
```

Comment:
So, when I read this, tests no longer run at all without saucelabs. Is that right?